tools/scripts/artifacts/nx-project.ts

```diff
@@ -245,13 +245,27 @@ export class NxProject {
   }
 
   public getPathToProjectInDist(): string {
+    const nestedPath = this.getProjectNestedPathFromWorkspaceJson()
+    const subPath = nestedPath ? nestedPath : path.join(this.nxProjectKind === NxProjectKind.Application
+      ? 'apps' : 'libs', this.name);
     return path.resolve(
       'dist',
-      this.nxProjectKind === NxProjectKind.Application ? 'apps' : 'libs',
-      this.name
+      subPath
     );
   }
 
+  public getProjectNestedPathFromWorkspaceJson(): string {
+    let workspaceJson = '';
+    try {
+      workspaceJson = fs.readFileSync("workspace.json").toString();
+      const workspaceJsonParsed = JSON.parse(workspaceJson);
```

Comment:
Unfortunately this won't work. Newly created workspaces, don't have a workspace.json